platform/api/_partials/resources/runners/reference.mdx

```diff
@@ -5844,7 +5844,7 @@ merge patch.
 
 ##### `optional` <span className="config-field-required" data-required="false">required</span> <span className="config-field-type">boolean</span> <span className="config-field-default"></span> <span className="config-field-enum"></span> <span data-pro="false" className="config-field-pro">pro</span> {#spec-clusterRef-podTemplate-spec-volumes-projected-sources-clusterTrustBundle-optional}
 
-If true, don't block pod startup if the referenced ClusterTrustBundles
+If true, don't block pod startup if the referenced ClusterTrustBundle(s)
```

Comment:
Could you please change here and in all other occurrences of noun(s) to just plural form?